package.json

```diff
@@ -38,6 +38,7 @@
     "path": "^0.12.7",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-modal": "^3.11.2",
```

Comment:
remind me to ask you about this dependencies later, I'm curious to know what it does. Good job!